mountpoint-s3/src/prefetch/part_queue.rs

```diff
@@ -84,13 +95,21 @@ impl<E: std::error::Error + Send + Sync> PartQueue<E> {
             Ok(part)
         }
     }
+
+    pub fn bytes_arrived_total(&self) -> usize {
```

Comment:
nit: if we're sticking with the naming above, then `total_bytes_arrived`

Reply:
chosen `bytes_received` as a less wordy alternative